@uportal/api-template-vue/src/components/ApiTemplateVue.vue

```diff
@@ -65,6 +65,10 @@ export default {
           headers: {
             Authorization: `Bearer ${token}`,
             'content-type': 'application/jwt'
+          },
+          validateStatus: function(status) {
```

Comment:
```suggestion
          validateStatus: function() {
```

Reply:
Hey Drew,
Agreed with @ChristianMurphy , believe me if for some reasons status is `undefined` in JavaScript, you may have `status` be something like `"undefined"`... JS is typed..., but in another way than java, C#,
suggest you to remove `status` here.

Reply:
No problem -- it's just my Java brain that expects these arguments to be listed... used or not!